doc/classes/ProjectSettings.xml

```diff
@@ -2658,6 +2658,21 @@
 		</member>
 		<member name="rendering/rendering_device/staging_buffer/texture_upload_region_size_px" type="int" setter="" getter="" default="64">
 		</member>
+		<member name="rendering/rendering_device/vsync/frame_queue_size" type="int" setter="" getter="" default="2">
+			The number of frames to track on the CPU side before stalling to wait for the GPU.
+			1 may give you the lowest lag/latency but at the high cost of no parallelism between CPU &amp; GPU.
+			Try the [url=https://darksylinc.github.io/vsync_simulator/]V-Sync Simulator[/url], an interactive interface that simulates presentation to better understand how it is affected by different variables under various conditions.
+			[b]Note:[/b] This property is only read when the project starts. There is currently no way to change this value at run-time.
+		</member>
+		<member name="rendering/rendering_device/vsync/swapchain_image_count" type="int" setter="" getter="" default="3">
+			The number of images the swapchain will consist of (back buffers + front buffer).
+			[code]2[/code] corresponds to double-buffering and [code]3[/code] to triple-buffering. A value of [code]1[/code] is not recommended.
+			Double-buffering may give you the lowest lag/latency but if V-Sync is on and the system can't render at 60 fps, the framerate will go down in multiples of it (e.g. 30 fps, 15, 7.5, etc.). Triple buffering gives you higher framerate (specially if the system can't reach a constant 60 fps) at the cost of up to 1 frame of latency, with [constant DisplayServer.VSYNC_ENABLED] (FIFO).
+			Use double-buffering with [constant DisplayServer.VSYNC_ENABLED]. Triple-buffering is a must if you plan on using [constant DisplayServer.VSYNC_MAILBOX] mode.
+			Try the [url=https://darksylinc.github.io/vsync_simulator/]V-Sync Simulator[/url], an interactive interface that simulates presentation to better understand how it is affected by different variables under various conditions.
+			[b]Note:[/b] This property is only read when the project starts. There is currently no way to change this value at run-time.
+			[b]Note:[/b] Some platforms may restrict the actual value.
```

Comment:
This note is a bit too up to interpretation. It would be lovely to mention a few platforms that do this, by name.

Reply:
Since the platforms and API backends are evolving over time, it's hard to say.

I know for sure these values are ignored in OpenGL (compatibility). But the other backends (D3D12, Metal) are not finished yet. It's also unclear what happens in macOS via MoltenVK.

Reply:
In that case, if the compatibility renderer is guaranteed to stay like this, that's worth mentioning (it also encompasses other platforms like Web).

Reply:
Rendering Device is not used by the compatibility renderer, which is where this setting lives on in the configuration panel. It doesn't sound like mentioning Compatibility renderer is relevant in that case to me.

The actual values that might be restricted by platforms is a pretty mixed bag to give clear examples and seems to change every now and then. I'd suggest a note that just says the value is a suggestion to the OS but that it might not match what the user actually requests at the low level.